blockchain/subscription.go

```diff
@@ -43,7 +43,7 @@ func (m *MockSubscription) GetEvent() *Event {
 func (m *MockSubscription) Close() {
 }
 
-/////////////////////////
+// ///////////////////////
```

Comment:
What is that?

Reply:
I honestly have no clue 😄 

Removed:

[1fa3f31](https://github.com/0xPolygon/polygon-edge/pull/675/commits/1fa3f318a648e5d8462831dac77970cc966a56a9)